NodeJS/sample/SampleAUXCodeClient.ts

```diff
@@ -0,0 +1,60 @@
+import * as Constants from "../src/Constants"
+import { createAUXCodeClient, AUXCodeClient } from "../src/AUXCodeClient"
+import { getValue } from "../src/Utils"
+
+const args = require('minimist')(process.argv.slice(2));
+
+let endpoint: string
+let apiKey: string
+
+try {
+    endpoint = getValue(Constants.ENDPOINT_KEY, args)
+    apiKey = getValue(Constants.API_KEY, args)
+    main()
+    .catch(error => console.error(error))
+    .finally(() => process.exit(-1))
+} catch (error) {
+    console.log(error)
+    process.exit(-1)
+}
+
+async function getAUXCodes(auxCodeClient: AUXCodeClient) {
+    try {
+        console.log("getAUXCodes calling");
+        return await auxCodeClient.getAuxCodes();
+    } catch (e) {
+        console.log("getAUXCodes error ");
+        console.error(e)
```

Comment:
You can club console.log() inside console.error() itself instead of two lines e.g. console.log("getAUXCodes error ", e)